app/src/main/kotlin/com/wire/android/di/CoreLogicModule.kt

```diff
@@ -391,6 +391,14 @@ class UseCaseModule {
     fun provideMarkGuestLinkFeatureFlagAsNotChangedUseCase(@KaliumCoreLogic coreLogic: CoreLogic, @CurrentAccount currentAccount: UserId) =
         coreLogic.getSessionScope(currentAccount).markGuestLinkFeatureFlagAsNotChanged
 
+//    @ViewModelScoped
+//    @Provides
+//    fun provideAppLockTeamFeatureConfigObserver(
+//        @KaliumCoreLogic coreLogic: CoreLogic,
+//        @CurrentAccount currentAccount: UserId
+//    ): AppLockTeamFeatureConfigObserver =
+//        coreLogic.getSessionScope(currentAccount).appLockTeamFeatureConfigObserver
+
```

Comment:
🤔 would this be waiting for Kalium's PR to be merged?

Reply:
Not needed in this PR, I will remove it